File: src/main/java/leets/leenk/domain/media/domain/service/MediaUpdateService.java
package leets.leenk.domain.media.domain.service;

import leets.leenk.domain.media.domain.entity.Media;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class MediaUpdateService {

    public void update(Media media, String thumbnailUrl){
        media.updateThumbnailUrl(thumbnailUrl);
    }
}

File: src/main/kotlin/leets/leenk/domain/media/domain/service/MediaUpdateService.kt
package leets.leenk.domain.media.domain.service

import leets.leenk.domain.media.domain.entity.Media
import org.springframework.stereotype.Service

@Service
class MediaUpdateService {
    fun update(
        media: Media,
        thumbnailUrl: String,
    ) {
        media.updateThumbnailUrl(thumbnailUrl)
    }
}
